### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by callmeadi** [redis/go-redis#2988] [General]
Hi, The last tagged release was in Feb. When do we expect the next official release to get the bug fixes from master branch?

**Comment by AzureMarker** [comment]
@monkey92t @vmihailenco Can we have a release soon? I'm looking forward to using [`CredentialsProviderContext`](https://github.com/redis/go-redis/issues/2681).

**Reply by monkey92t** [reply]
> @monkey92t @vmihailenco Can we have a release soon? I'm looking forward to using [`CredentialsProviderContext`](https://github.com/redis/go-redis/issues/2681).

OK, https://github.com/redis/go-redis/releases/tag/v9.5.2
